.github/workflows/labeler.yml

```diff
@@ -0,0 +1,72 @@
+# .github/labeler.yml
+
+# Label PRs that modify documentation files
+Documentation:
+  - changed-files:
+      - 'docs/**'
+
+# Label PRs with 'feature' if the head branch name starts with 'feature'
+Feature:
```

Comment:
Ensure that the regex for head-branch accurately matches your desired feature branches. Consider expanding this to include other branch naming conventions if necessary.